doc/appendix-de-kasssichv/appendix-de-kasssichv.md

```diff
@@ -94,12 +94,92 @@ The ftReceiptCase indicates the receipt type and defines how it should be proces
 For Germany (DE) the country code is `0x4445`. Thus, the value for an unknown ftReceiptCase in Germany is `0x4445000000000000`.
 
 | **Value** | **Description** | **Service- Version** |
-|----------------------|
+|----------------------|-----------------------------------------------------------------------------------------------------------------------------------|---------------------|
+| `0x4445000000000000` | "default value"<br />unknown payment type: automatic processing through the fiskaltrust.SecurityMechanisms settings is attempted. | 0-                  |
+| `0x4445000000000000` | "unknown payment type for DE"<br />This is handled like a Start Receipt.                                                          | 0-                  |
+| `0x4445000000000001` | "Start Transaction"                                                                                                               | 0-                  |
+| `0x4445000000000002` | "Update Transaction"                                                                                                              | 0-                  |
+| `0x4445000000000003` | "Delta Transaction"                                                                                                               | 0-                  |
+| `0x4445000000000004` | "End Transaction"                                                                                                                 | 0-                  |
+| `0x4445000000000005` | "Start Receipt"                                                                                                                   | 0-                  |
+| `0x4445000000000006` | "Stop Receipt"                                                                                                                    | 0-                  |
+| `0x4445000000000007` | "Daily Closing Receipt"                                                                                                           | 0-                  |
```

Comment:
please remove this until we defined businesscases

Reply:
done